### ISSUE TITLE
[ Request for Rename Template Pokemon ]

### ISSUE THREAD

**Issue by evetaku** [NoxxDev/NecroBot#747]
Rename with another format pokemon_IV example : Arcanine_100, its good when open in pogo and sorting by A-Z,because so difficult to find pokemon when sorting by A-Z with format 100_Arcanine.

Thanks

**Comment by ntatoo** [comment]
@evetaku 
"RenamePokemon": false, <<< Set True
"RenameOnlyAboveIv": false, <<< Set True
"RenameTemplate": "{1}_{0}",

**Comment by evetaku** [comment]
yes i know and use it, but that's format is IV_pokemon and in WIki "Format used by NecroBot to rename Pokemon. No need to modify."

i just request for another format, to easy find pokemon with format A-Z

**Comment by Whistler212** [comment]
"RenamePokemon": false, <<< Set True
"RenameOnlyAboveIv": false, <<< Set True
**"RenameTemplate": "{0}_{1}",**

**Comment by evetaku** [comment]
@Whistler212 really its work ? thanks, want to try

but pokemon with already renamed, cant renamed again . just new pokemon right ?

**Comment by Whistler212** [comment]
All pokemon on filter, even if it is already renamed

**Comment by Brazuca87** [comment]
The idea of having the numbers in front of the name, is so that you can sort by name and have basically a "sort by IV" in-game. If you want to see the pokemons together, you would sort by # instead.

But as Whistler212 said, just change the template in your config.

**Comment by MrProblematic** [comment]
There's any othe kind of rename template? Like, put attack, defense or stamina? ex: Arcanine with Atk: 10 Def: 8 Sta:15 => Rename him to> 10_8_15

**Comment by Brazuca87** [comment]
@MrProblematic None currently implemented.

**Comment by evetaku** [comment]
after change "RenameTemplate": "{0}_{1}", not working. no one renamed in template "RenameTemplate": "{0}_{1}",
